lib/Controller/RecordingController.php

```diff
@@ -100,4 +100,39 @@ public function store(string $owner): DataResponse {
 		}
 		return new DataResponse();
 	}
+
+	/**
+	 * @NoAdminRequired
+	 * @RequireModeratorParticipant
+	 */
+	public function notificationDismiss(int $timestamp): DataResponse {
+		try {
+			$this->recordingService->notificationDismiss(
+				$this->getRoom(),
+				$this->participant,
+				$timestamp
+			);
+		} catch (InvalidArgumentException $e) {
+			return new DataResponse(['error' => $e->getMessage()], Http::STATUS_BAD_REQUEST);
+		}
+		return new DataResponse();
+	}
+
+	/**
+	 * @NoAdminRequired
+	 * @RequireModeratorParticipant
+	 */
+	public function shareToChat(int $fileId, int $timestamp): DataResponse {
+		try {
+			$this->recordingService->shareToChat(
+				$this->getRoom(),
+				$this->participant,
+				$fileId,
+				$timestamp
+			);
+		} catch (InvalidArgumentException $e) {
```

Comment:
Should log the exception for better debugging?

Reply:
I think not because we already returned the error message on body of request. Do you prefer with log?

Reply:
I commented in the wrong place.